data/projects.js

```diff
@@ -212,6 +215,7 @@ projects = [
         "tags" : ["Vulture", "AST", "Bears"],
         "markdown": "vulture.md",
         "collaborating_projects" : ["coala", "vulture"]
```

Comment:
![screenshot from 2017-06-27 16-22-39](https://user-images.githubusercontent.com/21020094/27584266-eb1abac4-5b54-11e7-9d95-13ddcc392fd3.png)
Here's the screenshot you requested earlier with the developer's involved field modified for a project.